frontend/packages/knative-plugin/src/components/traffic-splitting/__tests__/TrafficSplittingFields.spec.tsx

```diff
@@ -30,15 +30,61 @@ describe('TrafficSplittingFields', () => {
         .first()
         .props().disableDeleteRow,
     ).toBe(true);
+    expect(
+      wrapper
+        .find(MultiColumnField)
+        .first()
+        .props().disableAddRow,
+    ).toBe(false);
   });
 
-  it('should not disable delete row button for more than one values', () => {
-    const wrapper = shallow(<TrafficSplittingFields {...formProps} />);
+  it('should not disable delete row button or add row button if number of values if more than one but less than total number of revisions', () => {
+    const wrapper = shallow(
+      <TrafficSplittingFields
+        {...formProps}
+        values={{
+          trafficSplitting: mockTrafficData
+            .filter((_, index) => index < 2)
+            .map((trafficElement) => ({ ...trafficElement, percent: 50 })),
+        }}
```

Comment:
IMO below will be simpler as it's mock data

```suggestion
        values={[
           { percent: 50, tag: 'tag-1', revisionName: 'overlayimage-fdqsf' },
           { percent: 50, tag: 'tag-2', revisionName: 'overlayimage-tkvz5' },
        ]}
```

Reply:
done